__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/development-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("200: returns all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const allTopics = response.body;
+        expect(allTopics).toHaveLength(3);
+        allTopics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
```

Comment:
Because this is quite a short array, you can check this by hard coding the array if you'd like.